src/service/plugins/mpris.js

```diff
@@ -344,7 +344,7 @@ var Plugin = new Lang.Class({
                 pos: Math.round( pos / 1000),
                 isPlaying: (player.PlaybackStatus === "Playing"),
                 canPause: (player.CanPause === true),
-                canPlay: (player.CanPlay === true),
+                canPlay: (player.CanPlay === true || player.PlaybackStatus === "Paused"),
```

Comment:
I think this should be filed as an upstream bug with VLC. `CanPlay` is a [mandatory property](https://specifications.freedesktop.org/mpris-spec/latest/Player_Interface.html#Property:CanPlay) of the MPRIS specification and implementing workarounds for specific players is not a path I want to go down. KDE Connect also uses [MPRIS in the almost the exact same way](https://github.com/KDE/kdeconnect-kde/blob/master/plugins/mpriscontrol/mpriscontrolplugin.cpp#L137-L207) (`CanPlay` means the method `Play()` can be invoked) so I think it's best to stick with that.

Reply:
Ok thanks I respect your decision